docs/src/app.tsx

```diff
@@ -19,7 +19,6 @@ class App extends React.Component<any, ThemeContextData> {
     return (
       <ThemeContext.Provider value={this.state}>
         <Provider
-          as={React.Fragment}
```

Comment:
Why this change is required? It will create an additional `div` element...

Reply:
because provider hasn't support for `as` - this has been detected by type checks:

```jsx
Provider extends React.Component<...>
```

Reply:
But it's supported by `ProviderBox`

Reply:
this should be resolved differently, as this case only will require us to open gate for any unhandled prop being passed to `Provider`. Let me see what could be done here